packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/widgets.dart';
+
+/// Interface for talking to the webview controller on the platform side.
+/// 
+/// An instance implementing this interface is passed to the `onWebViewCreated` callback that is
+/// passed to [WebViewPlatformInterface#onWebViewCreated].
+abstract class WebViewPlatformControllerInterface {
```

Comment:
nit: maybe "WebViewControllerPlatformInterface

Reply:
how about WebViewPlatform and FuchsiaWebViewPlatform/AndroidWebViewPlatform/etc?

Reply:
Ok I think II'm going with:
`WebViewPlatform`/`AndroidWebViewPlatform` and then `WebViewBuilder`/`AndroidWebViewBuilder`(instead of `WebViewPlatformInterface`)

This will mean to set it up for Fuchsia we will do:
`WebView.platformBuilder = FuchsiaWebViewBuilder();`

Reply:
Do WebViewBuilder and WebViewPlatform need to be distinct?

Reply:
We have a single `WebViewPlatform` per a platform webview instance, while a `WebViewBuilder` can create zero or more `WebViewPlatform` instances.

I don't see a nice way to merge them(other than adding a `webviewId` parameter to all the methods and then using a single instance of that merges `WebViewPlatform` to serve all of the webview instance).